lago_python_client/clients/group_client.py

```diff
@@ -16,7 +17,7 @@
     from typing import Mapping
 
 
-class GroupClient(BaseClient):
+class GroupClient(CreateCommandMixin, DestroyCommandMixin, FindCommandMixin, UpdateCommandMixin, BaseClient):
```

Comment:
The API only exposes a custom `find_all` route for groups, nested under `billable_metric` resource